ci/scripts/python_test.sh

```diff
@@ -47,9 +48,13 @@ export PYARROW_TEST_CUDA
 export PYARROW_TEST_DATASET
 export PYARROW_TEST_FLIGHT
 export PYARROW_TEST_GANDIVA
+export PYARROW_TEST_GCS
 export PYARROW_TEST_HDFS
 export PYARROW_TEST_ORC
 export PYARROW_TEST_PARQUET
 export PYARROW_TEST_S3
 
+# Without this, install_gcs_test_bench.sh doesn't seem to be put in a
+# place that the python env can find it.
+python -m pip install "https://github.com/googleapis/storage-testbench/archive/v0.16.0.tar.gz"
```

Comment:
I left a comment about this in `conftest.py` below.

Reply:
I changed this to check for the existence of the script and use the existing install comment.  I tried multiple approaches with docker but could get the environment variable set.